web/plugin-metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "quay-ui-plugin",
+  "version": "0.0.1",
+  "dependencies": {
+    "quay-ui": "^0.1.0"
+  },
+  "exposedModules": {
+    "QuayPluginMain": "./src/routes/PluginMain"
+  }
```

Comment:
You are missing an `extensions` list. Without it we won't know what to render.

@Hyperkid123 we should discuss how we want to handle multi route plugin. This plugin exposes similiar thing to what CRC app is exposing, however true plugin should expose just the routes and chroming service should handle the rendering.

Reply:
You don't need any extensions list to render in HCC. Scalprum is looking for the exposedModules directly and will be for some time. We can migrate to "true" plugin routing anyways until every HCC app is a plugin. This will suffice for some time.

Reply:
I'm gonna leave this as-is for now then